lib/lifecycle/properties-init.js

```diff
@@ -126,11 +126,11 @@
       }
 
       // We must coerce the initial value just in case it wasn't already.
-      info.internalValue = opts.coerce ? opts.coerce(initialValue) : initialValue;
+      var internalValue = info.internalValue = opts.coerce ? opts.coerce(initialValue) : initialValue;
```

Comment:
Visual studio code was complaining with `const` and a double assignment there so I played it safe.